data/src/main/scala/ch.epfl.scala.index.data/bintray/BintrayPipeline.scala

```diff
@@ -58,11 +57,6 @@ object BintrayPipeline {
       println("parent done")
     }
 
-    def claims(): Unit = {
-      val githubRepoExtractor = new GithubRepoExtractor(paths)
-      githubRepoExtractor.run()
-    }
```

Comment:
I removed this job that was modifying the `claims.json` file. This file is aimed to be read-only.